projects/storefrontapp-e2e-cypress/cypress/helpers/product-search-pricing-flow.ts

```diff
@@ -1,21 +1,17 @@
 import { PRODUCT_LISTING } from './data-configuration';
-import { checkFirstItem, createDefaultQueryRoute } from './product-search';
-import { apiUrl } from '../support/utils/login';
+import { createGenericQuery, createQuery } from './product-search';
 
 export function productPricingFlow() {
   cy.server();
-  createDefaultQueryRoute('query');
-  cy.route(
-    'GET',
-    `${apiUrl}/rest/v2/electronics-spa/products/search?fields=*&sort=price-asc*`
-  ).as('query_price_asc');
+  createGenericQuery('query');
+  createQuery('price-asc', 'query_price_asc');
 
   // Click on a Category
   cy.get('header').within(() => {
-    cy.get('.nav-link')
+    cy.get('cx-navigation-ui')
       .contains('Digital Cameras')
```

Comment:
Ran 5 headless (dev-17) tests and this fails 3 times

CypressError: Timed out retrying: cy.click() failed because this element is not visible:

`a tag role="link" ng-reflect-router-link="/Open-Catalogue/Cameras/Digita" href="/electronics-spa/en/USD/Open-Catalogue/Cameras/Digital-Cameras/Digital-Compacts/c/576">Compact...

This element a tag is not visible because it has an effective width and height of: '0 x 0' pixels.`

Reply:
This is working for me. Can you recheck?

Reply:
I'll recheck once dev-17 is back on, but I believe after that all is fine.

One key thing that I see is that this ticket is missing the other mobile tests that Bill mentioned for the tickets.
- mobile/cart
- mobile/added to cart modal 
were failing in yesterday's regression build

Reply:
yes, they are failing but there is another ticket for those.